### ISSUE TITLE
Hovering a stream under Favorites Category doesn't show stream title

### ISSUE THREAD

**Issue by jeovanny315** [MarkM-dev/Twitch-Previews#38]
Favorites:
https://i.imgur.com/23yGq0X.png

Followed:
https://i.imgur.com/uL3aQwg.png

If possible please add the title in favorites.

**Comment by MarkM-dev** [comment]
yep i know, somewhat problematic when the sidebar is closed but it's planned for sometime in the future.

**Comment by MarkM-dev** [comment]
Added in 3.7.
